src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -339,4 +340,21 @@ default GameModeData getGameModeData() {
      */
     void setSpectatorTarget(@Nullable Entity entity);
 
+    /**
+     * Gets the {@link WorldBorder} for this player, if present. If no border is
+     * set, an empty {@code Optional} is returned.
+     *
+     * @return The {@code WorldBorder} of this player as an {@code Optional}, if
+     * present
```

Comment:
Add 4 spaces indentation before the `present` to fix the [checkstyle warning](https://travis-ci.org/SpongePowered/SpongeAPI/jobs/222103695#L279):

```
src/main/java/org/spongepowered/api/entity/living/player/Player.java:348
Line continuation have incorrect indentation level, expected level should be 4.
[JavadocTagContinuationIndentation]
```